src/lang/builtins_time.ml

```diff
@@ -83,20 +76,33 @@ let () =
         ("isdst", Lang.bool tm.Unix.tm_isdst);
       ]
   in
-  add_builtin ~cat:Sys "localtime"
+  let nullable_time v =
+    match Lang.to_option v with
+      | Some v -> Lang.to_float v
+      | None -> Unix.gettimeofday ()
+  in
+  add_builtin ~cat:Sys "time.local"
     ~descr:
       "Convert a time in seconds into a date in the local time zone. Fields \
        meaning same as POSIX's `tm struct`. Warning: \"year\" is: year - 1900, \
-       i.e. 117 for 2017!" [("", Lang.float_t, None, None)] time_t (fun p ->
-      let tm = Unix.localtime (Lang.to_float (List.assoc "" p)) in
-      return tm);
-  add_builtin ~cat:Sys "gmtime"
+       i.e. 117 for 2017!"
```

Comment:
Could you update the description to mention that it either return current time or convert optional argument?